### PR TITLE
Add appveyor configuration [#180]

### PR DESCRIPTION
**Pull request by bezoerb**
You'll only need to create a free appveyor account and add your penthouse repository:
https://www.appveyor.com/docs/

Then you'll have penthouse tested on windows as well

### CODE REVIEW

**Comment by XhmikosR** [comment]
@pocketjoso: please look into adding AppVeyor.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - yarn install
+
+build: off
+
+test_script:
+  - node --version && npm --version
+  - yarn run test-all
+
+cache:
+  - "%LOCALAPPDATA%\\Yarn"
```
Comment (XhmikosR) [inline appveyor.yml]: Maybe add `-> yarn.lock`? Otherwise this cache entry will never be invalidated.

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - node_modules -> package.json
```
Comment (XhmikosR) [inline appveyor.yml]: I'd add `appveyor.yml` too in the files that can invalidate this cache entry.

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# https://www.appveyor.com/docs/appveyor-yml
+
+version: "{build}"
+
+clone_depth: 10
+
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "7"
```
Comment (XhmikosR) [inline appveyor.yml]: I'd remove 7 and add 10 here :)

**Comment by pocketjoso** [comment]
Closing due to inactivity. Open to pursuing this again in the future, if so we can re-open, or start a new PR.

**Pull request closed without merging**